weatherAplication/Modules/Home/View/HomeView.swift

```diff
@@ -0,0 +1,67 @@
+import SnapKit
+import UIKit
+
+class HomeView: UIView {
+    let kLogoWidth = 250
+    let kLogoHeight = 250
+    let kSideMargin = 10
+    let kTopMargin = 40
+    let kBottomMargin = 15
+    let kTableBottomMargin = 30
+
+    let logoImageView = UIImageView(image: R.image.weather_symbol())
+    let aplicationNameLabel = CustomLabel(text: R.string.localizable.my_weather())
+    let searchButton = SearchButton()
+    let lastCityNamesTableView = UITableView()
+
+    init() {
+        super.init(frame: .zero)
+        setupView()
+        setupConstraints()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func setupView() {
+        backgroundColor = .blue
+        addSubview(logoImageView)
+        addSubview(aplicationNameLabel)
+        lastCityNamesTableViewSetup()
+        addSubview(searchButton)
+    }
+
+    private func lastCityNamesTableViewSetup() {
+        lastCityNamesTableView.backgroundColor = .cyan
+        addSubview(lastCityNamesTableView)
+        lastCityNamesTableView.register(HomeTableViewCell.self, forCellReuseIdentifier: "cell")
```

Comment:
tutaj przenies stringa "cell" do stalej. spoko miejsce na to to np. HomeTableViewCell, gdzie tworzysz `static let kCellIdentifier = "cell"`, i mozesz sie do tego prosto odwolywac